src/libs/ReportUtils.ts

```diff
@@ -2523,6 +2515,27 @@ function getLinkedTransaction(reportAction: OnyxEntry<ReportAction | OptimisticI
     return allTransactions?.[`${ONYXKEYS.COLLECTION.TRANSACTION}${transactionID}`] ?? {};
 }
 
+/**
+ * Check if any of the transactions in the report has required missing fields
+ *
+ */
+function hasMissingSmartscanFields(iouReportID: string, isLHNPreview = false): boolean {
+    if (isLHNPreview) {
```

Comment:
Could you add a brief comment explaining why this distinction between LHNPreview and not is needed (and link the issue itself)

Reply:
Could you add this comment in code? Below the "Check if any of the transactions in the report has required missing fields" comment